pkg/schedule/scatter/region_scatterer_test.go

```diff
@@ -140,6 +146,7 @@ func scatter(re *require.Assertions, numStores, numRegions uint64, useRules bool
 		re.LessOrEqual(float64(count), 1.1*float64(numRegions)/float64(numStores))
 		re.GreaterOrEqual(float64(count), 0.9*float64(numRegions)/float64(numStores))
 	}
+	re.GreaterOrEqual(noNeedMoveNum, 0)
```

Comment:
It seems always be true

Reply:
yes, here only verify `noNeedMoveNum` is normal situation and no error reported.